Source/Charts/Components/AxisBase.swift

```diff
@@ -213,6 +213,16 @@ open class AxisBase: ComponentBase
     /// the total range of values this axis covers
     @objc open var axisRange = Double(0)
     
+    /// The minumum number of labels on the axis
+    @objc open var axisMinLabels: Int = Int(2) {
```

Comment:
Type inference is sufficient. `@objc open var axisMinLabels = 2`

Reply:
Fixed in new commit. Removed ```: Int``` but maintained ```Int(2)``` for consistence with the rest of the codebase.